src/Layers/xrRender_R2/r2_rendertarget_accum_reflected.cpp

```diff
@@ -67,7 +42,9 @@ void CRenderTarget::accum_reflected(light* L)
         RCache.set_Shader(s_accum_reflected);
 
         // Constants
-        RCache.set_c("Ldynamic_pos", L_pos.x, L_pos.y, L_pos.z, 1 / (L->range * L->range));
+        float att_R = L->range * .95f;
+        float att_factor = 1.f / (att_R * att_R);
+        RCache.set_c("Ldynamic_pos", L_pos.x, L_pos.y, L_pos.z, att_factor);
```

Comment:
This changes original behaviour. Please, remove this change and submit in a separate pull request along with direct light sources.